www/js/main.js

```diff
@@ -50,19 +40,320 @@ angular.module('emission.main', ['emission.main.diary', 'emission.main.common',
         controller: 'ControlCtrl'
       }
     }
+  })
+  .state('root.main.sensed', {
+    url: "/sensed",
+    views: {
+      'main-control': {
+        templateUrl: "templates/recent/sensedData.html",
+        controller: 'sensedDataCtrl'
+      }
+    }
+  })
+  .state('root.main.map', {
+      url: "/map",
+      views: {
+        'main-control': {
+          templateUrl: "templates/recent/map.html",
+          controller: 'mapCtrl'
+        }
+      }    
+  })
+  .state('root.main.log', {
+    url: '/log',
+    views: {
+        'main-control': {
+          templateUrl: 'templates/recent/log.html',
+          controller: 'logCtrl'
```

Comment:
These controllers don't have to be in the same file - they can also be in the a module that you include.